third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -105,6 +105,8 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
         .set_experiment(Experiment::SPEC_REWRITE, true)
         // Turn optimization on by default. Some configs below may turn it off.
         .set_experiment(Experiment::OPTIMIZE, true)
+        // TODO: disable after we have comparison testing
```

Comment:
Can we instead:
1. Create a new experiment called `OPTIMIZE_WAITING_FOR_COMPARE_TESTS`, which is set to `false` by default.
2. CFG simplification (and other new optimizations like the newer flush writes and future optimizations that need to undergo compare testing) derives from `OPTIMIZE_WAITING_FOR_COMPARE_TESTS`.
3. We can always set `OPTIMIZE_WAITING_FOR_COMPARE_TESTS` to true here, and in other places in transactional tests where you explicitly set CFG simplification experiment.
4. After compare testing for these, we can move these optimizations to instead derive from `OPTIMIZE` or `OPTIMIZE_FULL` (once Brian's optimize level PR lands).
5. This mechanism allows us to only change code at derivation point, instead of changing in multiple places for multiple in-flight optimizations.

If you would rather that I do it, let's land this first, so that I can do it in my flush writes optimization PR.

Reply:
done

Reply:
Can we please remove the TODO? This can always stay on, we would just derive those optimizations from "optimize" instead.

Reply:
done